app/assets/stylesheets/help-to-save-frontend.scss

```diff
@@ -21,6 +21,21 @@
     outline: 3px solid #ffbf47;
 }
 
+.button:focus {
```

Comment:
This will still apply to the 'button--link' buttons. Might be worth checking it looks ok still. If not, you can use :not(.button--link), to stop this style from applying.

Reply:
Looks to be okay
![HTS Accessibility](https://user-images.githubusercontent.com/37838443/70624829-044aea00-1c19-11ea-96d8-d44eb6df1bb5.gif)